### DISCUSSION TITLE
Migration from Airtable to NocoDB [Jun 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2419] [Discord Threads]
> 🙋‍♂️ **Ra**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@961503950778023956>, Thank you! Welcome to NocoDB!   
####    
#### > 1. Is Nocodb stable and reliable enough to start on right now or should we continue to use Airtable for a while (until Nocodb is feature complete and more stable) and then transfer to Nocodb?   
#### Yes. Many are using in production environment already. Explore, give us a spin. We are a small team working to ensure on wide range of feature sets. Any issue you find, let us know. We will be more than happy to help!   
####    
#### > 2. If we decide to use Nocodb, is Heroku deployment the way to go? Would it allow us to collaborate with the volunteers since it’s in the cloud? Can it be updated when new features are released?   
#### Heroku is one of the quick options available for hosting NocoDB. Collaboration with volunteers should be fine & you can upgrade NocoDB post every release.     
####    
#### > 3. What measures need to be taken to keep our database safe and secure?   
#### A periodic backup to start with; rest of the measures would be normal one's that you would have to do for any cloud hosted database.  
#  
> 🙋‍♂️ **Ra**  
#### Thank you <@857998597474484295>    
#### Following up on those answers: are any of the options other than Heroku compatible with team collaboration on a cloud? Looking for affordable / free options to begin with.  
#  
> 👨‍🦲 **dstala**  
#### You can explore Digital Ocean, Cloudron along with Heroku (there are many others too, you can discuss with discord community as well to see what comes out). All of these come with free tier options to start with. You will have to evaluate their pricing depending on your use case to narrow down what fits best.  
#  
> 💁 **o1lab**  
#### Once you set noco on any cloud, you can easily collaborate .. I would suggest to get a digital ocean droplet $5 a month.. you can run our docker compose  
#  
> 🙋‍♂️ **Ra**  
#### Thank you both. Will check them out 👍  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/984234350306557962)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
